.github/CODEOWNERS

```diff
@@ -30,6 +30,10 @@ book/ @ComposableFi/technical-writing-reviewers
 
 # Frontend and Blockchain Integration 
 frontend/ @ComposableFi/blockchain-integrations
+.github/workflows/frontend-pablo-eslint.yml @ComposableFi/blockchain-integrations
+.github/workflows/frontend-pablo-tests.yml @ComposableFi/blockchain-integrations
+.github/workflows/frontend-picasso-eslint.yml @ComposableFi/blockchain-integrations
+.github/workflows/frontend-picasso-tests.yml @ComposableFi/blockchain-integrations
```

Comment:
Would still want SRE to be an owner as well, as workflow files allow access to secrets.